patches/minecraft/net/minecraft/potion/Potion.java.patch

```diff
@@ -88,6 +88,18 @@
 +        ret.add(new net.minecraft.item.ItemStack(net.minecraft.init.Items.field_151117_aB));
 +        return ret;
 +    }
++    
++    /**
++     * Return an index that will override the potion's default sort order. This is useful for potions with colors 
++     * that change since, without this, the effect order would shift in the inventory GUI when stacked with others.
++     * 
++     * @return An index to override the default sort order
++     */
++    @SideOnly(Side.CLIENT)
```

Comment:
I just didn't see a need for it anywhere on the server. I guess it wouldn't hurt to make it server-accessible.